sommelier-wrapper

```diff
@@ -1,4 +1,5 @@
-#!/usr/bin/env -S bash -eu
+#!/usr/bin/env bash
```

Comment:
Would it be better to use `#!/bin/bash`?  That way if someone installs the crew bash package, we don't introduce potential unexpected issues.

Reply:
I don't think this is a showstopper...

Reply:
Changes made in #3 :)